pkg/proxy/proxy.go

```diff
@@ -158,11 +158,9 @@ func (s *SQLServer) onConn(ctx context.Context, conn net.Conn) {
 
 // Close closes the server.
 func (s *SQLServer) Close() error {
-	var errs []error
+	errs := make([]error, 0, 4)
```

Comment:
Why 4?

Reply:
Because there are four managers. So I just give a capacity of four.

Reply:
I'm afraid you misunderstood the comment. I didn't mean https://github.com/pingcap/TiProxy/pull/97/files#diff-104a11712684e520f0affa95e6a053faeb713306a10f69ee99b9729cc9dd1996R208 , I mean here.

Reply:
It is just a guess of possible connections, giving a larger initial start capacity.